lib/kafka_gen_stage/consumer.ex

```diff
@@ -141,7 +141,7 @@ defmodule KafkaGenStage.Consumer do
           consumer: pid(),
           consumer_ref: reference(),
           queue: :queue.queue(),
-          high_wm_offset: non_neg_integer() | atom(),
+          high_wm_offset: non_neg_integer(),
```

Comment:
I think `nil` still needs to be mentioned here. And in the type of `prepare_dispatch/4` as well.